modules/busco/main.nf

```diff
@@ -0,0 +1,82 @@
+process BUSCO {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::busco=5.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/busco:5.3.2--pyhdfd78af_0':
+        'quay.io/biocontainers/busco:5.3.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta, stageAs: 'tmp_input/*')  // Required:    meta map, and fasta sequence files
+    each lineage                                          // Required:    lineage to check against
+    path busco_lineages_path                              // Recommended: path to busco lineages - downloads if not set
+    path config_file                                      // Optional:    busco configuration file
+
+    output:
+    tuple val(meta), path("*-busco"), emit: busco_dir
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}-${lineage}"
+    def busco_config = config_file ? "--config $config_file" : ''
+    def busco_lineage_dir = busco_lineages_path ? "--download_path ${busco_lineages_path}" : ''
```

Comment:
If I understand "path to busco lineages - downloads if not set" correctly, `busco_lineages_path` should be referring to a path that exists and contains the pre-downloaded data. In this case, let's tell Busco it shouldn't download anything 
 
```suggestion
    def busco_lineage_dir = busco_lineages_path ? "--offline --download_path ${busco_lineages_path}" : ''
```